Assets/Scripts/Player/Movement/PlayerMovement.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections;
+using FrameWork.Extensions;
+using FrameWork.GridSystem;
+using UnityEngine;
+using UnityEngine.Serialization;
+
+namespace Player.Movement
+{
+    public class PlayerMovement : MonoBehaviour
+    { 
+        public float movementSpeed;
```

Comment:
```suggestion
        [SerializeField, Range(1,10) ] private float movementSpeed;
```
Je krijgt mooie balkje nu.